getting_started/5_Quantum_Teleportation.ipynb

```diff
@@ -0,0 +1,285 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "\n",
+    "# Simulate partial measurement by seperating out last Qubit\n",
```

Comment:
"Simulate partial measurement by seperating out last Qubit" suggested change: "Extract the measurement results of qubit 2 from the measurement counts for all three qubits"

Reply:
Updated